docker/irods_clients/minideb/bullseye/README.md

```diff
@@ -0,0 +1 @@
+An image with iRODS clients installed (iRODS icommands, baton suite)
```

Comment:
A description of the set up process might be useful here, e.g. 
Start the irods server with `docker run -d -p 1247:1247 <irods_image>` 
Run `path/to/start_client_container.sh`
Set up symlinks for required clients with `ln -s path/to/irods_client_wrapper.sh bin/iinit`

Reply:
I'm debating this. I'm intending for manual setup to be unnecessary by means of a Docker Compose configuration to do all of that.

Reply:
It's good to have this anyway - README improved now